docs/conf.py

```diff
@@ -116,6 +116,9 @@
 
 linkcheck_ignore = [
     'https://crates.io/crates/dredd-hooks',  # https://github.com/sphinx-doc/sphinx/pull/5140
+    # Exclude Dredd repository links, because they are checked
+    # as a part of another script.
```

Comment:
Would be nice to mention which one - `_extensions/ghlink_check.py`

Reply:
Mentioned in https://github.com/apiaryio/dredd/pull/1531/commits/5cc3afda02db6774f0256bb5de262335528f5905.